products/sle/main.pm

```diff
@@ -940,6 +940,7 @@ elsif (get_var('ISO_IN_EXTERNAL_DRIVE')) {
 }
 # post registration testsuites using suseconnect or yast
 elsif (have_scc_repos()) {
+    load_bootloader_s390x();    # schedule svirt/s390x bootloader if required
```

Comment:
I am sure that this fixes the problem for now but we should find a better way to schedule the necessary bootloader utility modules in a more generic way. Also, I do not think this is SLE-specific by definition. So I will merge this PR but we should try to improve it. I will talk to you about it.